<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ns::TestVoice</class>
 <widget class="QWidget" name="ns::TestVoice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>872</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>YesPo Service Client</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
	font-family: Verdana, &quot;宋体&quot;, Arial, Sans-serif, Segoe UI, Tahoma;
}

#widgetTestAudio {
	background: #ffffff;
	border-bottom : 1px solid #eff4f6;
}

#widgetTestAudio #labelSpeakImage {
	background-image: url(:/images/ui/test_audio_speaker.png);
	background-repeat: none;
	background-position: center center;
}

#widgetTestAudio #labelSpeakDescription {
	font-size: 15px;
	font-weight : bold;
}

QComboBox {
	border: 1px solid #d5cfcf;
	border-radius: 3px;
	padding: 1px 2px 1px 2px;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 24px;
    border-left-width: 1px;
    border-left-color: darkgray;
    border-left-style: solid; /* just a single line */
    border-top-right-radius: 3px; /* same radius as the QComboBox */
    border-bottom-right-radius: 3px;
}
 
QComboBox::down-arrow {
    image: url(:/images/ui/testvoice_down_arrow.png);
}

QComboBox::down-arrow:hover {
    image: url(:/images/ui/testvoice_down_arrow_hover.png);
}

QComboBox::down-arrow:pressed {
    image: url(:/images/ui/testvoice_down_arrow_pressed.png);
}

#widgetTestAudio #pushButtonSpeakTest {
	border-image: url(:/images/ui/test_playback.png) 0 3 0 3;
	border-top: 0px transparent;
	border-right: 3px transparent;
	border-bottom: 0px transparent;
	border-left: 3px transparent;
}

#widgetTestAudio #pushButtonSpeakTest:hover {
	border-image: url(:/images/ui/test_playback_hover.png) 0 3 0 3;
}

#widgetTestAudio #pushButtonSpeakTest:pressed {
	border-image: url(:/images/ui/test_playback_pressed.png) 0 3 0 3;
}

#widgetTestAudio #labelMicImage {
	background-image: url(:/images/ui/test_audio_microphone.png);
	background-repeat: none;
	background-position: center center;
}

#widgetTestAudio #labelMicDescription {
	font-size: 15px;
	font-weight : bold;
}

#widgetTestAudio #comboBoxMic {
	border: 1px solid #d5cfcf;
	border-radius: 3px;
}

#line {
	border : 1px solid #cccccc;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widgetTestAudio" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>24</number>
        </property>
        <property name="topMargin">
         <number>68</number>
        </property>
        <property name="bottomMargin">
         <number>50</number>
        </property>
        <item>
         <widget class="QLabel" name="labelSpeakTitle">
          <property name="text">
           <string>Speaker</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widgetSpeakerImage" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>64</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>64</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>2</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="labelSpeakImage">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelSpeakDescription">
          <property name="text">
           <string>Click the button, can you hear the sound?</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxSpeaker">
            <property name="minimumSize">
             <size>
              <width>190</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>190</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonSpeakTest">
            <property name="minimumSize">
             <size>
              <width>95</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>95</width>
              <height>32</height>
             </size>
            </property>
            <property name="text">
             <string>Test audio</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="minimumSize">
         <size>
          <width>1</width>
          <height>298</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>1</width>
          <height>298</height>
         </size>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <property name="spacing">
         <number>24</number>
        </property>
        <property name="topMargin">
         <number>68</number>
        </property>
        <property name="bottomMargin">
         <number>50</number>
        </property>
        <item>
         <widget class="QLabel" name="labelMicTitle">
          <property name="text">
           <string>Microphone</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widgetMicImage" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>64</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>64</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>6</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="labelMicImage">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelMicDescription">
          <property name="text">
           <string>Please speaking, can you hear your voice?</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxMic">
            <property name="minimumSize">
             <size>
              <width>190</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>190</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <spacer name="horizontalSpacer_7">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="ns::RenderArea" name="voiceDetectWidget" native="true">
            <property name="minimumSize">
             <size>
              <width>190</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>190</width>
              <height>20</height>
             </size>
            </property>
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_8">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ns::RenderArea</class>
   <extends>QWidget</extends>
   <header>voicewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="translator.qrc"/>
 </resources>
 <connections/>
</ui>
